rir/src/compiler/opt/inline.cpp

```diff
@@ -19,11 +19,12 @@ class TheInliner {
     TheInliner(Closure* function) : function(function) {}
 
     void operator()() {
```

Comment:
Is fuel the proper name here? I personally do not know that terminology. Inlining_threshold or something like that?
btw, it may be more tidy to squash this into another commit.

Reply:
I am not sure how standard it is, but I did not invent it myself.. the idea is that the optimizer has some budget to optimize and it can apply optimizations until it runs out of fuel.